ha/core/system_health/model/entity_health.py

```diff
@@ -151,6 +160,7 @@ def read(current_health: str):
                                                    current_health_dict[key][num_events]["created_timestamp"],
                                                    current_health_dict[key][num_events]["status"],
                                                    current_health_dict[key][num_events]["specific_info"])
+                        # TODO: Read method should not write new event
```

Comment:
Please explain more about this TODO.

Reply:
This is 
```
read()
```
read method so 
`entity_health.add_event(entity_event)`
should be separated from this method 

I tried this method to read old event but then it start writing new event

Reply:
If you see the method description it is for:
"Converts the entity health json into an object of this class."

So, I think this TODO is not applicable.

Reply:
`Converts the entity health json into an object of this class.`
Yes it should convert to object but not write new data to consul

Reply:
As discussed, please remove this todo.

Reply:
Removed todo